maps/ministation/uplink_item.dm

```diff
@@ -20,7 +20,7 @@
 	..()
 	item = null // Disabled
 
-/datum/uplink_item/stealthy_tools/syndigolashes/New()
+/datum/uplink_item/stealthy_tools/syndigaloshes/New()
```

Comment:
If it was in map files/vaults, wouldn't it cause a compile failure now that their path no longer exists?